datasetinsights/configs/faster_rcnn_fine_tune.yaml

```diff
@@ -42,3 +42,8 @@ optimizer:
 pretrained: False
 pretrained_backbone: True
 synchronize_metrics: True
+mlflow:
```

Comment:
If I didn't have mlflow section in my config yaml, what will happen? This seems to require all future yaml to include mlflow section which is not desirable.
- In current implementation this is required, but I can add some validation handle this. 
When we add enable disable feature this can be optional. 

Also, is this config required for every estimator yaml? Do they need to be repeated for every estimator yaml? Can they be shared across estimators?
- If an estimator wants to use MLFlow, he need MLFlow details. As of now we have decided to put MLFlow infra details in config so it need to be in all configs if its used by estimator. 

- It can be shared. I can create separate YAML file which is specific to MLFlow infra. All config need to import this if they want to use MLFlow.

Reply:
Since the yaml file with mlflow section will not be required in the future, they should be disabled by default. I still think it's valuable to implement NullTracker so that it's initialized by default when user does not specify mlflow in their yaml file.

Reply:
also, for consistency, we should probably rename this as the `tracker` or `tracking` section and there can be settings inside for mlflow, tensorboard, comet, wandb in the future